test/cookbooks/octopus-deploy-test/recipes/tentacle.rb

```diff
@@ -49,3 +49,44 @@
   checksum tentacle['checksum']
   trusted_cert '324JKSJKLSJ324DSFDF3423FDSF8783FDSFSDFS0'
 end
+
+# Install the Tentacle With User
+
+tentacle_with_user_dir = 'C:\Octopus2'
+
+directory tentacle_with_user_dir do
+  action :create
+end
+
+tentacle_with_user_certfile = File.join(tentacle_with_user_dir, 'tentacle_cert.txt')
+cookbook_file tentacle_with_user_certfile do
+  action :create
+  source 'cert.txt'
+end
+
+user node['octopus_deploy_test']['username'] do
+  password '5up3rR@nd0m'
+end
+
+# Install it here
+octopus_deploy_tentacle 'install TentacleWithUser' do
+  action :install
+  instance 'TentacleWithUser'
+  version tentacle['version']
+  checksum tentacle['checksum']
+end
+
+octopus_deploy_tentacle 'configure TentacleWithUser' do
+  action :configure
+  instance 'TentacleWithUser'
+  version tentacle['version']
+  checksum tentacle['checksum']
+  trusted_cert '324JKSJKLSJ324DSFDF3423FDSF8783FDSFSDFS0'
+  service_user ".\\#{node['octopus_deploy_test']['username']}"
+  service_password '5up3rR@nd0m'
+
```

Comment:
No, when creating a service the user needs to be fully qualified, either with the domain or machine name. The `.` is shorthand for the local machine. I'll make the user and password local variables